packages/generator/src/batch/type.ts

```diff
@@ -34,15 +34,12 @@ function getWriteRequestType(service: VdmServiceMetadata): string {
     e =>
       `CreateRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT> | UpdateRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT> | DeleteRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT>`
   );
-  const functionImportsReturnTypes = service.functionImports
-    .filter(fn => fn.httpMethod.toLowerCase() !== 'get')
+  const operationImportsReturnTypes = service.operationImports
```

Comment:
[nth] Generally, let's try to avoid the wording "operationIMPORT" and use "operation" instead. This was a misunderstanding initially.